_data/master/navbars/getting-started.yml

```diff
@@ -20,20 +22,6 @@ toc:
       path: /getting-started/kubernetes/managed-public-cloud/iks
     - title: Microsoft Azure Kubernetes Service (AKS)
       path: /getting-started/kubernetes/managed-public-cloud/aks
-  - title: Installation
-    section:
-    - title: Introduction
-      path: /getting-started/kubernetes/installation/
-    - title: Installing Calico for policy and networking (recommended)
-      path: /getting-started/kubernetes/installation/calico
-    - title: Installing Calico for policy (advanced)
-      path: /getting-started/kubernetes/installation/other
-    - title: Enabling application layer policy for Istio
-      path: /getting-started/kubernetes/installation/app-layer-policy
-    - title: Customizing the manifests
-      path: /getting-started/kubernetes/installation/config-options
-    - title: Integration guide (advanced)
-      path: /getting-started/kubernetes/installation/integration
```

Comment:
Why have all these links been removed?  It looks like `getting-started/kubernetes/installation/other.md` was removed, but all the rest of the docs still exist.

Reply:
Customizing the manifest is moved to Reference (per Casey decision)
Integration Guide is removed -- it was replaced by Calico the hard way and was never removed (per Casey).